app/client/rest/files.ts

```diff
@@ -76,13 +77,17 @@ const ClientFiles = (superclass: any) => class extends superclass {
             },
             timeoutInterval: toMilliseconds({minutes: 3}),
         };
+        if (!file.localPath) {
+            throw new Error('file does not have local path defined');
```

Comment:
is this error being handled?

Reply:
I haven't checked. My guess is that it is being handled along any other potential throw this function may have (like client calls).